README.md

```diff
@@ -15,9 +15,53 @@ Check the [issues](http://github.com/Chewsterchew/DIRCord/issues) to see what I'
 
 ## Setup guide.
 
```

Comment:
Put a ### Requirements here, indicate the requirements being Ruby (>= 2.4) and Git